.github/workflows/refresh_voc.yml

```diff
@@ -3,11 +3,12 @@ name: Refresh vocabulary tables
 on:
 #  schedule:
 ##    - cron: '0 0 * * 0'  # weekly, sunday 12am
-#    - cron: '0 6 * * *'  # every day 6am GMT (1-2am EST/EDT)
+#    - cron: '0 8 * * *'  # every day 6am GMT (3-4am EST/EDT)
```

Comment:
## Actions updates
- Counts/voc each run separately
- Voc runs 2 hours after counts, just in case